core/controllers/payload_validator.py

```diff
@@ -130,6 +130,7 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
     'ImageUploadHandler',
     'IncomingReplyEmailHandler',
     'InstantFeedbackMessageEmailHandler',
+    'InternetConnectivityHandler',
```

Comment:
Please do add the schema for the handler. You can use this [wiki](https://github.com/oppia/oppia/wiki/Writing-schema-for-handler-args) to add the schema. Thanks.

Reply:
This handler does not require the schema. This handler is similar to ```FrontendErrorHandler```  and handles only a single get request to check that server is connected to the frontend. I added this handler to ```HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS```.

Reply:
Thank you for the clarification.

Reply:
>> This handler does not require the schema. This handler is similar to FrontendErrorHandler and handles only a single get request to check that server is connected to the frontend. I added this handler to HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS.

Doesn't feel like a sufficient explanation. I think schema validation can be done.
```
    URL_PATH_ARGS_SCHEMAS = {}
    HANDLER_ARGS_SCHEMAS = {
        'GET': {}
    }
```

Reply:
Done.